pages/k8s/release-notes.md

```diff
@@ -13,6 +13,19 @@ layout: [base, ubuntu-com]
 toc: False
 ---
 
+# CVE-2018-18264
```

Comment:
@evilnick Do you think we should we start adding a date to these headers so it's clear when the release happened?

Reply:
Yes!

Reply:
Yeah, but I prefer January 10, 2019 or 2019-01-10.